{"primaryContentSections":[{"kind":"declarations","declarations":[{"languages":["swift"],"platforms":["macOS"],"tokens":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","preciseIdentifier":"s:SS","text":"String"},{"text":"? { ","kind":"text"},{"text":"get","kind":"keyword"},{"kind":"text","text":" }"}]}]}],"variants":[{"traits":[{"interfaceLanguage":"swift"}],"paths":["\/documentation\/postgresclientorm\/tableobjecterror\/errordescription"]}],"abstract":[{"text":"Inherited from ","type":"text"},{"type":"codeVoice","code":"LocalizedError.errorDescription"},{"type":"text","text":"."}],"kind":"symbol","hierarchy":{"paths":[["doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM","doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError"]]},"sections":[],"schemaVersion":{"minor":3,"patch":0,"major":0},"metadata":{"role":"symbol","modules":[{"name":"PostgresClientORM"}],"roleHeading":"Instance Property","title":"errorDescription","externalID":"s:17PostgresClientORM16TableObjectErrorO16errorDescriptionSSSgvp","symbolKind":"property","fragments":[{"text":"var","kind":"keyword"},{"text":" ","kind":"text"},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"kind":"typeIdentifier","text":"String","preciseIdentifier":"s:SS"},{"text":"?","kind":"text"}]},"identifier":{"url":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/errorDescription","interfaceLanguage":"swift"},"references":{"doc://PostgresClientORM/documentation/PostgresClientORM":{"role":"collection","type":"topic","abstract":[],"kind":"symbol","title":"PostgresClientORM","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM","url":"\/documentation\/postgresclientorm"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError/errorDescription":{"url":"\/documentation\/postgresclientorm\/tableobjecterror\/errordescription","role":"symbol","identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError\/errorDescription","abstract":[],"kind":"symbol","fragments":[{"text":"var","kind":"keyword"},{"kind":"text","text":" "},{"text":"errorDescription","kind":"identifier"},{"kind":"text","text":": "},{"text":"String","preciseIdentifier":"s:SS","kind":"typeIdentifier"},{"text":"?","kind":"text"}],"type":"topic","title":"errorDescription"},"doc://PostgresClientORM/documentation/PostgresClientORM/TableObjectError":{"navigatorTitle":[{"text":"TableObjectError","kind":"identifier"}],"role":"symbol","fragments":[{"text":"enum","kind":"keyword"},{"text":" ","kind":"text"},{"kind":"identifier","text":"TableObjectError"}],"identifier":"doc:\/\/PostgresClientORM\/documentation\/PostgresClientORM\/TableObjectError","url":"\/documentation\/postgresclientorm\/tableobjecterror","type":"topic","kind":"symbol","abstract":[],"title":"TableObjectError"}}}